src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -525,7 +532,7 @@ trait BeamHelper extends LazyLogging {
     }
     logger.warn(logStart)
 
-    val injector: inject.Injector = buildInjector(config, beamExecutionConfig.beamConfig, scenario, beamScenario)
+    val injector = buildInjector(config, beamExecutionConfig.beamConfig, scenario, beamScenario)
```

Comment:
Why remove annotation - this helps compile time, so if there we should keep there.